cmd/agent/subcommands/subcommands_nix.go

```diff
@@ -13,8 +13,8 @@ import (
 	cmdstop "github.com/DataDog/datadog-agent/cmd/agent/subcommands/stop"
 )
 
-// windowsSubcommands returns SubcommandFactories for subcommands dependent on the `windows` build tag.
-func windowsSubcommands() []command.SubcommandFactory {
+// lifecycleSubcommands returns SubcommandFactories for subcommands to stop the agent on Unix.
+func lifecycleSubcommands() []command.SubcommandFactory {
```

Comment:
I'm curious why this remains a distinct file?

Reply:
I was on the fence on this. For the other it's a matter of features, here it's a matter of OS.
I felt that having a unix version of `subcommands/controlsvc` or an empty `stop` subcommand for windows would not make much sens.

But I'm not 100% convinced too, WDYT ?

Reply:
My gut says to do it the same way for everything, just to avoid future devs having to think about which approach to take for a new subcommand.